src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -149,10 +149,12 @@ public VectorizeConfig(
             Map<String, String> authentication,
             Map<String, Object> parameters) {
           this.provider = provider;
-          // HuggingfaceDedicated does not need user to specify model
-          // use endpoint-defined-model as placeholder
+          // HuggingfaceDedicated does not need user to specify model explicitly
+          // If user specifies modelName other than endpoint-defined-model, will error out
+          // By default, huggingfaceDedicated provider use endpoint-defined-model as placeholder
           if (provider.equals(ProviderConstants.HUGGINGFACE_DEDICATED)) {
-            if (modelName != null) {
+            if (modelName != null
+                && !modelName.equals(ProviderConstants.HUGGINGFACE_DEDICATED_DEFINED_MODEL)) {
               throw ErrorCode.INVALID_CREATE_COLLECTION_OPTIONS.toApiException(
                   "'modelName' is not needed for provider %s",
```

Comment:
Should this mention what the "default" model name (which is accepted is)

Reply:
Indeed, Tatu.
Let me add, real quick.